.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
[EOF](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)
https://eslint.org/docs/rules/eol-last